src/Education/XamlGenerator/IRuleHelpXamlBuilder.cs

```diff
@@ -61,13 +64,8 @@ internal partial class RuleHelpXamlBuilder : IRuleHelpXamlBuilder
         public FlowDocument Create(IRuleHelp ruleHelp)
         {
             var xaml = CreateXamlString(ruleHelp);
-
-            using (var stringReader = new StringReader(xaml))
-            {
-                var xamlReader = XmlReader.Create(stringReader);
-                var flowDocument = (FlowDocument)XamlReader.Load(xamlReader);
-                return flowDocument;
-            }
+            var flowDocument = (FlowDocument)XamlReader.Parse(xaml);
+            return flowDocument;
```

Comment:
Miscellaneous: `XamlReader.Parse` handles strings, so we don't need to create a reader ourselves.